packages/components/src/Icon/_docs/icon.stories.scss

```diff
@@ -16,9 +16,9 @@
   display: flex;
   flex-direction: column;
   align-items: center;
-  justify-content: center;
-  gap: var(--spacing-16);
-  padding: var(--spacing-8);
+  justify-content: flex-start;
+  gap: var(--spacing-8);
+  padding: var(--spacing-16);
```

Comment:
Perhaps debatable, but I feel like the balance seems off when I look at the new styles. I see they're more aligned though, but the spacing gaps look a bit wrong (likely related to the fixed width/height).

Maybe if you're playing with visuals, using `display: grid` might be better than flex :)

Reply:
I've updated it a bit using grid and removed the set height. It still looks a little off when you have a row where some break over multiple lines and some don't but I'd rather they all start at the same reading position than jump around like they use to.

## before
![Screenshot 2024-03-12 at 10 49 48 am](https://github.com/cultureamp/kaizen-design-system/assets/36558508/05cb7ac0-885a-49e8-81eb-eed4c116b506)

## after
![Screenshot 2024-03-12 at 10 50 28 am](https://github.com/cultureamp/kaizen-design-system/assets/36558508/3c76c917-0e35-4b5d-98ac-bca9923b6d3a)

Reply:
Much better 😄 I think it was just the uneven spacing caused by the set height which was throwing me off :)